src/main/java/panda/Panda.java

```diff
@@ -0,0 +1,61 @@
+package panda;
+import panda.command.Command;
+import panda.component.Parser;
+import panda.component.Storage;
+import panda.component.TaskList;
+import panda.component.Ui;
+import panda.exception.PandaException;
+
+public class Panda {
+
+    private TaskList tlist;
```

Comment:
Would it be better to have a more descriptive name such as taskList or simply tasks?